# HG changeset patch
# User Rik <rik@octave.org>
# Date 1510884078 28800
#      Thu Nov 16 18:01:18 2017 -0800
# Branch stable
# Node ID 4746bd294c54aeb2f6e3fb40372531ca9abcd730
# Parent  ec4ec2f7029d73fb3a42fb581641ef2932528c01
Fix definition of Delaunay triangulation in docstrings (bug #52416).

* delaunay3.m, delaunay.m: Fix definition of Delaunay triangulation in
docstring.

diff --git a/scripts/deprecated/delaunay3.m b/scripts/deprecated/delaunay3.m
--- a/scripts/deprecated/delaunay3.m
+++ b/scripts/deprecated/delaunay3.m
@@ -20,17 +20,17 @@
 ## @deftypefn  {} {@var{tetr} =} delaunay3 (@var{x}, @var{y}, @var{z})
 ## @deftypefnx {} {@var{tetr} =} delaunay3 (@var{x}, @var{y}, @var{z}, @var{options})
 ##
 ## @code{delaunay3} is deprecated and will be removed in Octave version 4.4.
 ## Please use @code{delaunay} in all new code.
 ##
 ## Compute the Delaunay triangulation for a 3-D set of points.
 ## The return value @var{tetr} is a set of tetrahedrons which satisfies the
-## Delaunay circum-circle criterion, i.e., only a single data point from
+## Delaunay circum-circle criterion, i.e., no data point from
 ## [@var{x}, @var{y}, @var{z}] is within the circum-circle of the defining
 ## tetrahedron.
 ##
 ## The set of tetrahedrons @var{tetr} is a matrix of size [n, 4].  Each
 ## row defines a tetrahedron and the four columns are the four vertices
 ## of the tetrahedron.  The value of @code{@var{tetr}(i,j)} is an index into
 ## @var{x}, @var{y}, @var{z} for the location of the j-th vertex of the i-th
 ## tetrahedron.
diff --git a/scripts/geometry/delaunay.m b/scripts/geometry/delaunay.m
--- a/scripts/geometry/delaunay.m
+++ b/scripts/geometry/delaunay.m
@@ -19,30 +19,30 @@
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{tri} =} delaunay (@var{x}, @var{y})
 ## @deftypefnx {} {@var{tetr} =} delaunay (@var{x}, @var{y}, @var{z})
 ## @deftypefnx {} {@var{tri} =} delaunay (@var{x})
 ## @deftypefnx {} {@var{tri} =} delaunay (@dots{}, @var{options})
 ## Compute the Delaunay triangulation for a 2-D or 3-D set of points.
 ##
 ## For 2-D sets, the return value @var{tri} is a set of triangles which
-## satisfies the Delaunay circum-circle criterion, i.e., only a single data
-## point from [@var{x}, @var{y}] is within the circum-circle of the defining
-## triangle.  The set of triangles @var{tri} is a matrix of size [n, 3].  Each
-## row defines a triangle and the three columns are the three vertices of the
-## triangle.  The value of @code{@var{tri}(i,j)} is an index into @var{x} and
-## @var{y} for the location of the j-th vertex of the i-th triangle.
+## satisfies the Delaunay circum-circle criterion, i.e., no data point from
+## [@var{x}, @var{y}] is within the circum-circle of the defining triangle.
+## The set of triangles @var{tri} is a matrix of size [n, 3].  Each row defines
+## a triangle and the three columns are the three vertices of the triangle.
+## The value of @code{@var{tri}(i,j)} is an index into @var{x} and @var{y} for
+## the location of the j-th vertex of the i-th triangle.
 ##
 ## For 3-D sets, the return value @var{tetr} is a set of tetrahedrons which
-## satisfies the Delaunay circum-circle criterion, i.e., only a single data
-## point from [@var{x}, @var{y}, @var{z}] is within the circum-circle of the
-## defining tetrahedron.  The set of tetrahedrons is a matrix of size [n, 4].
-## Each row defines a tetrahedron and the four columns are the four vertices of
-## the tetrahedron.  The value of @code{@var{tetr}(i,j)} is an index into
-## @var{x}, @var{y}, @var{z} for the location of the j-th vertex of the i-th
+## satisfies the Delaunay circum-circle criterion, i.e., no data point from
+## [@var{x}, @var{y}, @var{z}] is within the circum-circle of the defining
+## tetrahedron.  The set of tetrahedrons is a matrix of size [n, 4].  Each row
+## defines a tetrahedron and the four columns are the four vertices of the
+## tetrahedron.  The value of @code{@var{tetr}(i,j)} is an index into @var{x},
+## @var{y}, @var{z} for the location of the j-th vertex of the i-th
 ## tetrahedron.
 ##
 ## The input @var{x} may also be a matrix with two or three columns where the
 ## first column contains x-data, the second y-data, and the optional third
 ## column contains z-data.
 ##
 ## The optional last argument, which must be a string or cell array of strings,
 ## contains options passed to the underlying qhull command.
